specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-03-01/virtualMachine.json

```diff
@@ -1099,6 +1099,20 @@
             "type": "string",
             "description": "The system query option to filter VMs returned in the response. Allowed value is 'virtualMachineScaleSet/id' eq /subscriptions/{subId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachineScaleSets/{vmssName}'"
           },
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The expand expression to apply on operation. 'instanceView' enables fetching run time status of all Virtual Machines, this can only be specified if a valid $filter option is specified",
+            "enum": [
+              "instanceView"
+            ],
+            "x-ms-enum": {
+              "name": "ExpandTypesForGetVMs",
```

Comment:
suggest naming this "ExpandTypesForListVMs" instead. because there is a enum name that is for Get VMs https://github.com/Azure/azure-rest-api-specs/blob/e5aeda96c7154543fca8dc475e5214f36d516859/specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-03-01/virtualMachine.json#L824

Reply:
Resolved